src/app/Providers/ComposeServiceProvider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\ServiceProvider;
+use Illuminate\Support\Facades\View;
+
+class ComposeServiceProvider extends ServiceProvider
+{
+    /**
+     * Register any application services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        //
+    }
+
+    /**
+     * Bootstrap any application services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        View::composer('*', function($view) {
+            if(auth()->check()) {
+                
+                $name = auth()->user()->name;
+                $view->with('name', $name);
+            }
```

Comment:
個人的には、こっちのほうが拡張性あるから（複数の値を指定できるから）好き笑
```suggestion
            if(auth()->check()) {
                $name = auth()->user()->name;
                $view->with(compact('name'));
            }
```